deployment.rb

```diff
@@ -76,8 +76,8 @@ def lib_dir(*dirs)
   deploy_dir('lib', *dirs)
 end
 
-def bin_i18n_dir(*dirs)
-  deploy_dir('bin/i18n', *dirs)
+def bin_dir(*dirs)
+  deploy_dir('bin', *dirs)
```

Comment:
why do we want to deploy more directories?

Reply:
I didn't quite understand the question

Reply:
What does this change (to this file in general) do?

Reply:
It's just a shortcut for the `bin/i18n` folder that we currently use only for running tests.
https://github.com/code-dot-org/code-dot-org/pull/52481/files#diff-4ceb4fa7fd06129c4ebffd2ed8283aea5bac6f4db02713795bda233ce3591cf5L93

I removed it since we use `bin_dir` instead:
https://github.com/code-dot-org/code-dot-org/pull/52481/files#diff-4ceb4fa7fd06129c4ebffd2ed8283aea5bac6f4db02713795bda233ce3591cf5R93